src/components/map/map.tsx

```diff
@@ -171,9 +194,31 @@ export class Map implements GxComponent {
     }
   }
 
+  private setUserLocation({ coords }) {
+    this.userLocationCoords = `${coords.latitude}, ${coords.longitude}`;
+    this.userLocationChange.emit(this.userLocationCoords);
+    console.log("this.userLocationCoords", this.userLocationCoords);
+  }
+
+  componentWillLoad() {
+    console.log("WILL LOAD!");
+    if (this.watchPosition) {
+      this.watchPositionId = watchPosition(
+        this.setUserLocation.bind(this),
+
```

Comment:
This spaces seem off. I'd stick with Prettier's suggested styling.

Reply:
fixed in ab4b869